### DISCUSSION TITLE
上传文件到中国移动云盘，怎样保留文件的修改时间？

### DISCUSSION THREAD

**Discussion by gdtv** [AlistGo/alist#7114] [Q&A]
### 问题描述
用PC版的中国移动客户端上传文件，是可以保留文件的修改时间的。但是在alist网页和windows里通过webdav挂载alist进行上传，就不能保留文件的修改时间。

![屏幕截图 2024-08-30 162325](https://github.com/user-attachments/assets/e25d0670-807f-4888-9e40-9e84a8a5942e)

### 版本
```
./alist version
Built At: 2024-07-21 13:20:26 +0000
Go Version: go1.22.4 linux/amd64
Author: Xhofe <i@nn.ci>
Commit ID: d4e3355f
Version: v3.36.0
WebVersion: 3.36.0
```

### 驱动
移动云盘

**Comment by mailbyms** [comment]
用 rclone 同步上去，是可以保留时间的。

**Reply by Topkill** [reply]
请教一下该怎么做呢
